ccd-definition/State.json

```diff
@@ -5,46 +5,52 @@
     "ID": "Open",
     "Name": "Open",
     "Description": "Initial case state – create title as a minimum; add documents, etc.",
-    "DisplayOrder": 1
+    "DisplayOrder": 1,
+    "TitleDisplay": "# ${caseName}\n## **CCD ID: #${[CASE_REFERENCE]}**"
```

Comment:
This approach probably produces empty line if optional value is not present

Reply:
There is a validation on case name when we are submitting the case. So, BA confirmed its ok to have empty line for the open case if there is no case name